### ISSUE TITLE
The __defineGetter__ syntax is now deprecated per MDN

### ISSUE THREAD

**Issue by baki250** [angular/ngSocket#17]
Hi 
The **defineGetter** syntax is now deprecated per MDN.

Will you provide a hotfix to use .

Thanks

**Comment by Wildhoney** [comment]
Use [Socket.io](http://socket.io/) :thumbsup:

**Comment by jeffbcross** [comment]
Yep, this should use `defineProperty`

**Comment by baki250** [comment]
Yep done that, if you could update your codebase for others too, or i can merge my changes, whichever

**Comment by baki250** [comment]
@Wildhoney Unfortunately the backend system doesnt support node :(
